bookwyrm/forms.py

```diff
@@ -242,14 +243,39 @@ def clean(self):
         filetype = cleaned_data.get("filetype")
         book = cleaned_data.get("book")
         domain = urlparse(url).netloc
+
+        try:
+            response = requests.head(url, timeout=3)
+
+            if response.status_code in range(300, 308):
+                headers = response.headers
+                url_redirect = headers["Location"]
+                print(url_redirect)
+
+                response = requests.head(url_redirect, timeout=3)
+            if int(response.status_code) < 200 or int(response.status_code) > 299:
+                # status code between 200 and 399 should be fine
+                self.add_error(
+                    "url",
+                    _(
+                        f"""This url throws status code {response.status_code} and can't be added 
+                        to list"""
+                    ),
+                )
+        except:
+            self.add_error(
+                "url", _("This domain does not exist. Please check your entry.")
+            )
+
         if models.LinkDomain.objects.filter(domain=domain).exists():
             status = models.LinkDomain.objects.get(domain=domain).status
             if status == "blocked":
                 # pylint: disable=line-too-long
                 self.add_error(
                     "url",
                     _(
-                        "This domain is blocked. Please contact your administrator if you think this is an error."
+                        """This domain is blocked. Please contact your administrator 
```

Comment:
Did this get changed automatically? It ought to stay on one line, so that the translation string doesn't have unexpected whitespace